index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Leonardo Pareja Portfolio</title>
+</head>
+<body>
+  <header class="block">
+    <h1 class="name">Leonardo Pareja</h1>
+    <img class="menu" src="images/icons/Menu_icon.svg" alt="menu icon">
+  </header>
+  <main class="block container">
+    <section class="section01">
+      <h2 class="title title01 items">
+        Hey there. <br>I'm Leonardo
+      </h2>
+      <h3 class="title text01 items">I'm a Software Developer</h3>
+      <p class="text text02 items">I can help you build a product, 
+        feature or website look through some of my work and 
+        experience! if you like what you see and have a project 
+        you need coded, don't hesitate to contact me.</p>
+        <div class="social_media items">
+          <img src="images/icons/github_icon.svg" alt="github_icon" class="icon icon01">
+          <img src="images/icons/Linkedin_icon.svg" alt="Linkedin_icon" class="icon icon02">
+          <img src="images/icons/hand_icon.svg" alt="hand_icon" class="icon icon03">
+          <img src="images/icons/twitter_icon.svg" alt="twitter_icon" class="icon icon04">
+          <img src="images/icons/M_icon.svg" alt="M_icon" class="icon icon05">
+        </div>
```

Comment:
- [ ] I would suggest wrapping these icons inside the `<ul>` tag (unordered list) because this is the common convention when listing elements. Also, don't forget to use the `<li>` tag (list item) for each element inside the list. [Here you can read and see a few examples about it](https://www.w3schools.com/tags/tag_ul.asp) :hugs:

  ![image](https://user-images.githubusercontent.com/81522435/138315610-ad362f33-79a7-4d81-900a-72f5cb9aad95.png)

- [ ] Also, these images should be links to your social networks. Please use the `<a>` for these images. Each list item should look like: `<li><a href="SOCIAL_URL"><img src="./Images/..." alt="Icon"></a></li>` :memo: